modules/my_health/app/controllers/my_health/v1/prescriptions_controller.rb

```diff
@@ -138,6 +160,36 @@ def collection_resource
           client.get_active_rxs_with_details
         end
       end
+
+      def set_filter_metadata(list)
```

Comment:
just leaving this comment here, we already talked about moving this function `set_filter_metadata()` and the ones associated to it to the helper file, however we had some problems trying that, given our time restraint, well move it on a separate PR.